openeo/local/processing.py

```diff
@@ -6,22 +6,31 @@
 from pathlib import Path
 
 from openeo_pg_parser_networkx import ProcessRegistry
-from openeo_processes_dask.core import process as openeo_process
+from openeo_processes_dask.process_implementations.core import process
+from openeo_pg_parser_networkx.process_registry import Process
 
-standard_processes = [
+PROCESS_REGISTRY = ProcessRegistry(wrap_funcs=[process])
+
+# Import these pre-defined processes from openeo_processes_dask and register them into registry
+processes_from_module = [
     func
     for _, func in inspect.getmembers(
         importlib.import_module("openeo_processes_dask.process_implementations"),
         inspect.isfunction,
     )
 ]
 
-PROCESS_REGISTRY = ProcessRegistry(wrap_funcs=[openeo_process])
+specs_module = importlib.import_module("openeo_processes_dask.specs")
```

Comment:
Any particular reason you use `importlib.import_module("openeo_processes_dask.specs")`
instead of just `import openeo_processes_dask.specs` and just work with `openeo_processes_dask.specs` as an object?

Reply:
same actually with `importlib.import_module("openeo_processes_dask.process_implementations"),` above

Reply:
@LukeWeidenwalker I need your help here, this code comes from: https://github.com/Open-EO/openeo-processes-dask/blob/9afc89aca7f1090810572e4d8c42c0a450db02d5/tests/conftest.py#L57

Reply:
Ah, this was copied over from the process registry initialisation code in the executor where I did something like `importlib.import_module(f"{module_source}.specs")` to grab processes from different sources. Afaics this can be safely removed here as @soxofaan suggests!

Reply:
Fixed with the latest commit